File: examples-sdk/go/604-db-elastic-search/main.go
// 604-db-elastic-search — Go SDK equivalent
// Compile: vil compile --from go --input 604-db-elastic-search/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("app", 8080)
	s.Compile()
}

File: examples-sdk/csharp/604-db-elastic-search/Main.cs
// 604-db-elastic-search — C# SDK equivalent
// Compile: vil compile --from csharp --input 604-db-elastic-search/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();
